coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2360,12 +2360,12 @@ def test_reshape(self, backend, output_shape):
 
 class TestFlatten(TorchBaseTest):
     @pytest.mark.parametrize(
-        "backend, start_dim",
-        itertools.product(backends, [2,-2],),
+        "backend, start_dim, end_dim",
+        itertools.product(backends, [2, -2, 0], [3, -1]),
     )
-    def test_reshape(self, backend, start_dim):
+    def test_flatten(self, backend, start_dim, end_dim):
         input_shape = (2, 3, 4, 5)
-        model = ModuleWrapper(function=torch.flatten, kwargs={"start_dim": start_dim})
+        model = ModuleWrapper(function=torch.flatten, kwargs={"start_dim": start_dim, "end_dim": end_dim})
         self.run_compare_torch(input_shape, model, backend=backend)
```

Comment:
Thanks, we need to add the dynamic input unit test as well.
Please do the similar approach like https://github.com/apple/coremltools/blob/main/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py#L1106

Reply:
@fukatani 
You might need to do something like this:
```
        converter_input_type = [TensorType(shape=(1, 3, RangeDim(), RangeDim()), dtype=np.float32)]
        mlmodel = self.run_compare_torch(input_shape, model,
                               backend=backend,
                               converter_input_type=converter_input_type)[1]
```
change input_shape to RangeDim won't work I believe ...
Does this code actually pass on your local? :O